### PR TITLE
MaySpawnObjectNotFromPlugin

### PR DESCRIPTION
**Pull request by Klarulor**
I cant normally use object in my plugin. Conflict happens

### CODE REVIEW

**Comment by Ficus-x** [comment]
Yeeeeeep, merge into main

File: MapEditorReborn/Patches/UpdatePositionServerPatch.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ namespace MapEditorReborn.Patches
     [HarmonyPatch(typeof(AdminToyBase), nameof(AdminToyBase.UpdatePositionServer))]
     internal static class UpdatePositionServerPatch
     {
-        private static bool Prefix(AdminToyBase __instance) => false;
```
Comment (Michal78900) [inline MapEditorReborn/Patches/UpdatePositionServerPatch.cs:19]: This affects all AdminToys, not only ones that are a part of CustomSchematic

**Review by Michal78900 — CHANGES_REQUESTED** [review]

File: MapEditorReborn/Patches/UpdatePositionServerPatch.cs (hunk 1 above)
Comment (Michal78900) [reply MapEditorReborn/Patches/UpdatePositionServerPatch.cs:19]: Even before I overidded this bit of code as it wasn't working as it should, also I safe a little bit performance not calling 3 properties per frame per object

Comment (Klarulor) [reply MapEditorReborn/Patches/UpdatePositionServerPatch.cs:19]: Then how i can use my plugin with MapEditorReborn? I cant spawn a lightsource/networkspawner::spawn()

https://github.com/Klarulor/SCPSL-TorchPlayer/blob/main/EventHandlerscs.cs#L23

Comment (Michal78900) [reply MapEditorReborn/Patches/UpdatePositionServerPatch.cs:19]: If you are using MER as a reference you can use MER methods to spawn stuff
`ObjectSpawner.SpawnLight()` should give you the same result

Comment (Klarulor) [reply MapEditorReborn/Patches/UpdatePositionServerPatch.cs:19]: Then i cant spawn without ur MER, right?

Comment (Michal78900) [reply MapEditorReborn/Patches/UpdatePositionServerPatch.cs:19]: you can, but it may be affected even if you don't want that

Comment (Michal78900) [reply MapEditorReborn/Patches/UpdatePositionServerPatch.cs:19]: if you don't need to use MER, just don't use it and use your own methods

**Comment by Klarulor** [comment]
But i cant!  Object just isnt spawn

**Comment by Klarulor** [comment]
If i remote MER, then it works

**Pull request closed without merging**